### ISSUE TITLE
Startproject treats `dotgithub` as templates

### ISSUE THREAD

**Issue by sergei-maertens** [maykinmedia/default-app#8]
The github CI matrix expressions should be properly escaped and not evaluated as Django template code.

**Comment by Bartvaderkin** [comment]
I confirm this is an issue.

**Comment by sergei-maertens** [comment]
yeah the templates should probably use `{% templatetag openvar %}` instead, or perhaps they can be excluded from templating if that's not relevant at all

**Comment by sergei-maertens** [comment]
Was fixed in 9d949841
